samples/rest/bing_auto_suggest_v7.py

```diff
@@ -0,0 +1,71 @@
+"""Python example showcasing the usage of Bing Auto Suggest API using REST calls"""
+
+import json
+from pprint import pprint
+
+import dotenv
+import requests
+
+
+def auto_suggest_basic(
+    subscription_key,
+    auth_header_name="Ocp-Apim-Subscription-Key",
+    query="sail",
```

Comment:
I believe we shouldn't set a default value for the query since that's the essential parameter that the function operates on. Maybe move that value to the call in the main function.

Reply:
done